src/Examine.AzureDirectory/AzureLuceneDirectory.cs

```diff
@@ -169,7 +237,55 @@ public override void TouchFile(string name)
             CacheDirectory.TouchFile(name);
             SetDirty();
         }
+        protected void SyncFile(BlobClient _blob, string fileName)
+        {
+            Trace.WriteLine($"INFO Syncing file {fileName} for {RootFolder}");
+            // then we will get it fresh into local deflatedName 
+            // StreamOutput deflatedStream = new StreamOutput(CacheDirectory.CreateOutput(deflatedName));
+            using (var deflatedStream = new MemoryStream())
```

Comment:
`deflatedStream` here isn't the correct name, it's just stream that the blob is downloaded too. I realize this was a copy/paste from other code but should be renamed, even just `memStream` or whatever is fine.